docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/event-specifications/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Event specifications"
+sidebar_position: 115
+---
+
+# Event Specifications integration
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+The plugin allows you to integrate with Event Specifications for a selected set of plugins. The configuration for the plugin should be retrieved directly from your [Data Product](https://docs.snowplow.io/docs/understanding-your-pipeline/data-products/) in the [Snowplow BDP Console](https://console.snowplowanalytics.com).
+
+The plugin will automatically add an Event Specification context to the events matching the configuration added. 
+
+:::note
+The plugin is available since version 3.23 of the tracker.
+:::
+
+## Install plugin
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+| Tracker Distribution | Included |
+|----------------------|----------|
+| `sp.js`              | ❌        |
+| `sp.lite.js`         | ❌        |
+
+**Download:**
+
+<table><tbody><tr><td>Download from GitHub Releases (Recommended)</td><td><a href="https://github.com/snowplow/snowplow-javascript-tracker/releases">Github Releases (plugins.umd.zip)</a></td></tr><tr><td>Available on jsDelivr</td><td><a href="https://cdn.jsdelivr.net/npm/@snowplow/browser-plugin-event-specifications@latest/dist/index.umd.min.js">jsDelivr</a> (latest)</td></tr><tr><td>Available on unpkg</td><td><a href="https://unpkg.com/@snowplow/browser-plugin-event-specifications@latest/dist/index.umd.min.js">unpkg</a> (latest)</td></tr></tbody></table>
+
+```javascript
+window.snowplow(
+    'addPlugin',
+    'https://cdn.jsdelivr.net/npm/@snowplow/browser-plugin-event-specifications@latest/dist/index.umd.min.js',
+    ['eventSpecifications', 'EventSpecificationsPlugin']
+);
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+- `npm install @snowplow/browser-plugin-event-specifications`
+- `yarn add @snowplow/browser-plugin-event-specifications`
+- `pnpm add @snowplow/browser-plugin-event-specifications`
+
+
+```javascript
+import { newTracker } from '@snowplow/browser-tracker';
+import { WebVitalsPlugin } from '@snowplow/browser-plugin-event-specifications';
+
+newTracker('sp1', '{{collector_url}}', { 
+   appId: 'my-app-id', 
+   plugins: [ EventSpecificationsPlugin(/* plugin configuration */) ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+## Configuration
+
+The configuration of the plugin would be in the form of:
+
+```ts
```

Comment:
Can we also give instructions how to configure the plugin when using the JS tag? It's not super intuitive, there is a [general guide here](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/configuring-tracker-plugins/javascript/#plugins-after-initialization), but better to have it as part of the plugin docs I think.